mysql-test/main/outfile_fifo-master.sh

```diff
@@ -0,0 +1,5 @@
+if [ -e $MYSQLTEST_VARDIR/tmp/fifo ]; then
+    rm $MYSQLTEST_VARDIR/tmp/fifo
+fi
+mkfifo $MYSQLTEST_VARDIR/tmp/fifo
+cat $MYSQLTEST_VARDIR/tmp/fifo > $MYSQLTEST_VARDIR/tmp/fifo_data.txt && touch $MYSQLTEST_VARDIR/tmp/done.txt &
```

Comment:
```suggestion
if [ -e "$MYSQL_TMP_DIR"/fifo ]; then
    rm "$MYSQL_TMP_DIR"/fifo
fi
if [ -e "$MYSQL_TMP_DIR"/done.txt ]; then
    rm "$MYSQL_TMP_DIR"/done.txt
fi
mkfifo "$MYSQL_TMP_DIR"/fifo
cat "$MYSQL_TMP_DIR"/fifo > "$MYSQL_TMP_DIR"/fifo_data.txt && touch "$MYSQL_TMP_DIR"/done.txt &
```

Using MYSQL_TMP_DIR and removing done.txt